ogcapi-stable/ogcapi-collections/src/main/java/de/ii/ogcapi/collections/app/CollectionsBuildingBlock.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2022 interactive instruments GmbH
+ *
+ * <p>This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy
+ * of the MPL was not distributed with this file, You can obtain one at
+ * http://mozilla.org/MPL/2.0/.
+ */
+package de.ii.ogcapi.collections.app;
+
+import com.github.azahnen.dagger.annotations.AutoBind;
+import de.ii.ogcapi.collections.domain.CollectionsConfiguration;
+import de.ii.ogcapi.collections.domain.ImmutableCollectionsConfiguration;
+import de.ii.ogcapi.foundation.domain.ExtensionConfiguration;
+import de.ii.ogcapi.foundation.domain.ApiBuildingBlock;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * @title Feature Collections (COLLECTIONS)
+ * @en The module Feature Collections has to be enabled for every API with a feature provider.
+ * It provides the resources Feature Collections and Feature Collection. Currently feature
+ * collections are the only supported type of collection.
+ * Feature Collections implements all requirements of conformance class Core of
+ * OGC API - Features - Part 1: Core 1.0 for the two mentioned resources.
+ * |Resource |Path |HTTP Method |Media Types | --- | --- | --- |Feature Collections |
+ * /{apiId}/collections |GET |JSON, HTML, XML |Feature Collection
+ * |/{apiId}/collections/{collectionId}|GET |JSON, HTML, XML
+ * @de Das Modul "Feature Collections" ist für jede über ldproxy bereitgestellte API mit einem
+ * Feature-Provider zu aktivieren. Es stellt die Ressourcen "Collections" und "Collection" bereit.
+ * Derzeit sind Feature Collections die einzige unterstütze Art von Collections.
+ * "Feature Collections" implementiert alle Vorgaben der Konformitätsklasse "Core" von
+ * OGC API - Features - Part 1: Core 1.0 für die zwei genannten Ressourcen.

```

Comment:
@CabukEge You have to copy the raw markdown (see https://raw.githubusercontent.com/interactive-instruments/ldproxy/master/docs/de/configuration/services/building-blocks/collections.md), so that for example links are preserved. 
This applies to all files.